app/routes/bdb/utils.tsx

```diff
@@ -29,7 +29,7 @@ export const NonEventContactStatusConfig: Record<
   [NonEventContactStatus.NOT_INTERESTED]: {
     displayName: 'Ikke interessert',
     color: 'var(--danger-color)',
-    textColor: '#ff0000',
+    textColor: '#fff',
```

Comment:
Could we maybe do the `--color-absolute-white`? I know this is the same, but I prefer to use our variable in case we want to change them